lms/templates/certificates/invalid.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
 <%inherit file="accomplishment-base.html" />
 <%! from django.utils.translation import ugettext as _ %>
```

Comment:
Line 15 in this file contains an aria-label attribute that should probably be translated.